src/main/java/command/Command.java

```diff
@@ -0,0 +1,43 @@
+package command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import exception.DukeException;
+
+/**
+ * Parent Abstract Class to be inherited by actual children classes
+ */
+public abstract class Command {
+    private CommandType type;
+
+    Command(CommandType type) {
+        this.type = type;
+    }
+
+    /**
+     * Runs the command, modifying the contents of taskList, Ui or storage depending on the
+     * implementation of child class
+     *
+     * @param taskList ArrayList of Tasks Objects
+     * @param ui       Object of the Ui class
+     * @param storage  Object of the Storage class
+     * @throws DukeException Exception that occurs while executing the command
```

Comment:
Should there be a punctuation behind each parameter description? I noticed the same issue in several other Javadoc comments too.